### PR TITLE
Add EKS instructions for open-iscsi

### PR DESCRIPTION
**Pull request by meldafrawi**
Signed-off-by: meldafrawi <mohamed.eldafrawi@rancher.com>

### CODE REVIEW

**Comment by yasker** [comment]
Typo in the title. Also, don't need the quote for the second part. Just add it to the end of the previous sentence.

**Comment by meldafrawi** [comment]
@yasker fixed.

File: README.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ Longhorn is 100% open source software. Project source code is spread across a nu
     1. For GKE, recommended Ubuntu as guest OS image since it contains open-iscsi already.
     2. For Debian/Ubuntu, use `apt-get install open-iscsi` to install.
     3. For RHEL/CentOS, use `yum install iscsi-initiator-utils` to install.
+    4. For EKS with `EKS Kubernetes Worker AMI with AmazonLinux2 image`, 
+       use `yum install iscsi-initiator-utils` to install. you may need to edit cluster security group to allow ssh access
```
Comment (yasker) [inline README.md]: `You` instead of `you`. And ending with dot.

**Comment by meldafrawi** [comment]
longhorn/longorn@1086

**Pull request merged**